s3repo/view.py

```diff
@@ -0,0 +1,90 @@
+"""View for working with the repositories on S3."""
+
+import os
+
+from flask import jsonify
+from flask import render_template
+from flask import Response
+from flask import request
+from flask.views import View
+
+
+class S3View(View):
+    """View for working with S3 according to the REST model."""
+
+    def __init__(self, model):
+        self.model = model
+
+    @staticmethod
+    def response_message(message, status):
```

Comment:
I think it might be private: `_response_message`

Reply:
Removed.